src/tools/tidy/src/deps.rs

```diff
@@ -131,6 +131,7 @@ const WHITELIST: &[Crate<'_>] = &[
     Crate("polonius-engine"),
     Crate("ppv-lite86"),
     Crate("proc-macro2"),
+    Crate("punycode"),
```

Comment:
Why is this now needed?

Reply:
We didn't check it before? It's presumably only depended on by librustc_codegen_lvm